extensions/exposition/source/directives/octets/Store.ts

```diff
@@ -141,12 +152,29 @@ export class Store implements Directive {
 
     return await this.discovery[key]
   }
+
+  private parseMeta (value: string | string[]): Record<string, string> {
+    if (Array.isArray(value))
+      value = value.join(',')
+
+    const meta: Record<string, string> = {}
+
+    for (const pair of value.split(',')) {
+      const [k, v] = pair.trim().split('=')
```

Comment:
Let's add `limit` parameter to `.split` to allow `=` in values (i.e. base64 encoded things)

```suggestion
      const [k, v] = pair.trim().split('=', 2)
```

Reply:
Updated (`split` doesn't work this way).

Reply:
NIT: yeah, something like that would do:
```ts
const [k, v] = pair.trim().split(/=(.*)/s, 2)
```